supervisor/hardware/manager.py

```diff
@@ -103,7 +103,13 @@ def _import_devices(self) -> None:
         # Exctract all devices
         for device in self._udev.list_devices():
             # Skip devices without mapping
-            if not device.device_node or self.helper.hide_virtual_device(device):
+            try:
+                if not device.device_node or self.helper.hide_virtual_device(device):
+                    continue
+            except UnicodeDecodeError as err:
+                # Some udev properties have an unkown/different encoding. This is a general
+                # problem with pyudev, see https://github.com/pyudev/pyudev/pull/230
+                _LOGGER.warning("Ignoring udev device due to error: %s", err)
```

Comment:
printing a string that potentially is not decodable doesnt seem to be helpful in debugging.

Reply:
@RichardPar Sorry about that. We're trying out this AI code review bot for suggestions since the amount of PRs in our organization is quite high, much higher then the number of reviewers. The idea is that hopefully it helps with some good initial suggestions but I had thought its suggestions would either be accepted or rejected and that was that. I didn't expect it to do a bunch of follow-up insisting on its changes like this. I'll talk to the other HA owners to see if there's anything we can do to tone it down here.